scanpy/preprocessing/simple.py

```diff
@@ -354,7 +354,9 @@ def pca(
     data: Union[AnnData, np.ndarray, spmatrix],
     n_comps: int = N_PCS,
     zero_center: Optional[bool] = True,
+    sparse_pca: Optional[bool] = False,
```

Comment:
Maybe make it `None` by default, and, based on the benchmarks, use it when the data is sufficiently large.

In this case, make sure you document this in the parameter docs and log a debug message.